lib/build-view.js

```diff
@@ -17,6 +17,7 @@ module.exports = (function() {
     this.titleLoopIndex = 0;
     this.a2h = new Convert();
     this.monocle = false;
+    this.attach();
```

Comment:
But then if we remove this attach `this.height` will be uninitialized when `minimizedHeight` gets loaded from the config. 

The whole point was that we need to get the height of the panel while it still exists beacuse otherwise the height will be `0`. And since the panel might not have been ever made visible prior to reading `minimizedHeight` we need to explicitly create it somewhere.

Changing the order of `observe` calls might work too I guess, but this seemed more reliable and I wasn't sure if they are synchronous or not.